### PR TITLE
🌐 Add Portuguese translation for `docs/tutorial/query-params.md`

### PR DESCRIPTION
**Pull request by FelipeSilva93**
#1211

### CODE REVIEW

File: docs/en/mkdocs.yml [hunk 1]
```diff
@@ -163,9 +163,6 @@ markdown_extensions:
     permalink: true
 - markdown.extensions.codehilite:
     guess_lang: false
-# Uncomment these 2 lines during development to more easily add highlights
```
Comment (cassiobotaro) [inline docs/en/mkdocs.yml:166]: Por que está alterando estas linhas?

File: docs/pt/docs/tutorial/path-params.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+
+	Neste caso, a URL deveria ser: `/files//home/johndoe/myfile.txt`, com barra dupla (`//`) entre `files` e `home`.
+
+
+## Recapitulando
+
+Com o **FastAPI**, usando as declarações de tipo do Python, você obtém:
+
+* Suporte no editor: verificação de erros, e opção de autocompletar, etc.
+* Parsing de dados
```
Comment (cassiobotaro) [inline docs/pt/docs/tutorial/path-params.md:239]: Análise de dados?

**Review by diogoduartec — CHANGES_REQUESTED** [review]
**Actually you're translating the wrong page, the correct page is docs/tutorial/path-params.md.**

**Comment by LorhanSohaky** [comment]
Can we close it? 

related #4775

**Comment by tiangolo** [comment]
Thanks @LorhanSohaky!

As this was handled in another PR I'll close this one. ☕

**Pull request closed without merging**